src/GitHub.VisualStudio/GitHub.VisualStudio.csproj

```diff
@@ -17,6 +18,7 @@
     <VsixType>v3</VsixType>
     <IsProductComponent>false</IsProductComponent>
     <ExtensionInstallationFolder>GitHub\GitHub</ExtensionInstallationFolder>
+	<XlfLanguages>zh-CN</XlfLanguages>
```

Comment:
I notice you've specified `zh-CN` here rather than `zh-Hans` (which I think is simplified Chinese). Is the documentation on the https://github.com/dotnet/xliff-tasks README wrong?

> `XlfLanguages` - The set of locales to which the project is localized. Defaults to the thirteen locales supported by Visual Studio: `cs;de;es;fr;it;ja;ko;pl;pt-BR;ru;tr;zh-Hans;zh-Hant`.

Does Visual Studio support `zh-CN`, `zh-Hans` and `zh-Hant`? 😕

Reply:
I'm... not sure. The [original PR](https://github.com/github/VisualStudio/pull/1714) had `zh-HANS` translations, but then when it was re-done in #1888, the translations that came from crowdin were `zh-CN` and this seems to work fine.

As for which is correct, I'm afraid I don't know enough about internationalization or Chinese in particular to say. I've tried googling and I'm not the wiser ;)

As I say, it seems to work fine, so ¯\_(ツ)_/¯

Reply:
> https://twitter.com/jcansdale/status/1042490984892903424
> Is it okay to use zh-CN instead of zh-Hans/zh-Hant in Visual Studio?

The resource manager will probe based on CurrentUICulture from a culture through its parents:

For zh-CN, this goes zh-CN -> zh-CHS -> zh-Hans.

This means, for example that if CurrentUICulture is zh-CN, then it will use zh-Hans resources, but not vice versa. It might be the case that VS always sets CurrentUICulture to zh-CN when using Chinese (Simplified) language pack, in which case it would work to use zh-CN, but I am not certain of that. If that isn't true, then using zh-CN will not work in all cases.

In the end, there are just two possible choices for VS language packs:

![image](https://user-images.githubusercontent.com/75470/45788166-c0438200-bc2d-11e8-9255-4c57b1114ae0.png)

And their descriptions align to zh-Hans and zh-Hant
```
> using System.Globalization;
> new CultureInfo("zh-Hans").DisplayName
"Chinese (Simplified)"

> new CultureInfo("zh-Hant").DisplayName
"Chinese (Traditional)"
```

So I would suggest lining up with that and just using zh-Hans and zh-Hant.

Reply:
@maikebing when you submitted the latest round of translations from crowdin using `zh-CN` rather than `zh-Hans`, was there a reason for that?

Reply:
I'm afraid it doesn't appear to work when the `Chinese (Traditional)` language pack is installed. 😭 

![image](https://user-images.githubusercontent.com/11719160/45808836-af8f1c80-bcbe-11e8-816c-ef37834c3add.png)

Reply:
It appears that `CultureInfo.CurrentUICulture` is set to `zh-TW` when using the `Chinese (Traditional)` language pack. The parents of this appear to be `zh-TW -> zh-CHT -> zh-Hant -> zh`.

@maikebing am I correct in thinking that `zh-Hans` and `zh-Hant` are treated like completely separate languages? There is no mechanism for `zh-Hant` to fall back to using `zh-Hans`?